app.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-#cd Alembic
-#alembic revision --autogenerate -m "DB creation"
-#alembic upgrade head
-#cd ..
+cd Alembic
+alembic revision --autogenerate -m "DB creation"
+alembic upgrade head
+cd ..
```

Comment:
Try to use `Alembic/alembic revision --autogenerate -m "DB creation"` and `Alembic/alembic upgrade head` instead of `cd` if possible